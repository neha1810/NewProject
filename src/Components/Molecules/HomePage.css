.element-div{
    text-align: center;
}
.price{
    color: #f10102;
    font-size: 18px;
    font-weight: 700;
}
img{
    width:100%;
}

