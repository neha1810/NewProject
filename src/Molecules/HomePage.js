import React, { Component } from 'react'
import gift from '../Images/giftcard.png'
import amazonGift from '../Images/amazon-gift.jpg'
import '../Molecules/HomePage.css'

class homePage extends Component {
    render() {
        return (
            <div>
                <h1>Products</h1>
                <content>
                    <div className="container-fluid">
                        <div className="row">
                            <div className="col-xs-12 col-sm-4 col-md-4" >
                                <div className="element-div">
                                    <a href="/" className="card-link"> <img src={gift} alt="Gift Card" /></a>
                                </div>
                                <div className="details">
                                    <h2>Gift-Card</h2>
                                    <h3 className="price"> $99</h3>
                                </div>
                            </div>

                            <div className="col-xs-12 col-sm-4 col-md-4">
                                <div className="element-div">
                                    <a href="/" className="card-link"> <img src={gift} alt="Gift Card" /></a>
                                </div>
                                <div className="details">
                                    <h2>Gift-Card</h2>
                                    <h3 className="price"> $999</h3>
                                </div>
                            </div>

                            <div className="col-xs-12 col-sm-4 col-md-4">
                                <div className="element-div">
                                    <a href="/" className="card-link"><img src={amazonGift} alt="Gift Card" /></a>

                                </div>
                                <div className="details">
                                    <h2>Gift-Card</h2>
                                    <h3 className="price"> $500</h3>
                                </div>
                            </div>
                        </div>

                        <div className="row">
                            <div className="col-xs-12 col-sm-4 col-md-4" >
                                <div className="element-div">
                                    <a href="/" className="card-link"> <img src={gift} alt="Gift Card" /></a>
                                </div>
                                <div className="details">
                                    <h2>Gift-Card</h2>
                                    <h3 className="price"> $99</h3>
                                </div>
                            </div>
                            <div className="col-xs-12 col-sm-4 col-md-4">
                                <div className="element-div">
                                    <a href="/" className="card-link"> <img src={gift} alt="Gift Card" /></a>
                                </div>
                                <div className="details">
                                    <h2>Gift-Card</h2>
                                    <h3 className="price"> $999</h3>
                                </div>
                            </div>

                            <div className="col-xs-12 col-sm-4 col-md-4">
                                <div className="element-div">
                                    <a href="/" className="card-link"><img src={amazonGift} alt="Gift Card" /></a>
                                </div>
                                <div className="details">
                                    <h2>Gift-Card</h2>
                                    <h3 className="price"> $500</h3>
                                </div>
                            </div>
                        </div>
                        <div className="row">
                            <div className="col-xs-12 col-sm-4 col-md-4" >
                                <div className="element-div">
                                    <a href="/" className="card-link"> <img src={gift} alt="Gift Card" /></a>
                                </div>
                                <div className="details">
                                    <h2>Gift-Card</h2>
                                    <h3 className="price"> $99</h3>
                                </div>
                            </div>
                            <div className="col-xs-12 col-sm-4 col-md-4">
                                <div className="element-div">
                                    <a href="/" className="card-link"> <img src={gift} alt="Gift Card" /></a>
                                </div>
                                <div className="details">
                                    <h2>Gift-Card</h2>
                                    <h3 className="price"> $999</h3>
                                </div>
                            </div>

                            <div className="col-xs-12 col-sm-4 col-md-4">
                                <div className="element-div">
                                    <a href="/" className="card-link"><img src={amazonGift} alt="Gift Card" /></a>
                                </div>
                                <div className="details">
                                    <h2>Gift-Card</h2>
                                    <h3 className="price"> $500</h3>
                                </div>
                            </div>
                        </div>

                    </div>
                </content>
            </div>
        )
    }
}
export default homePage;