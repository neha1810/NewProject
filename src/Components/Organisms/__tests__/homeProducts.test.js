import React from 'react';
import {shallow} from 'enzyme';
import { configure } from 'enzyme';
import { Provider } from 'react-redux'
import { createStore, applyMiddleware } from 'redux';
import thunk from 'redux-thunk';
import rootReducer from '../../Redux/reducers'
import Adapter from 'enzyme-adapter-react-16';
import HomePage from '../homeImages/homeProducts';

const store = createStore(rootReducer, applyMiddleware(thunk));
configure({ adapter: new Adapter() });

describe('HomePage', () => {
let wrapper;
beforeEach(()=> {
wrapper = shallow(
        <Provider store={store}><HomePage /></Provider>
    )
})


it('render the HomePage component', () => {
    expect(wrapper).toMatchSnapshot();
});
it('Button exists',()=> {

    expect(wrapper.find('.btn btn-primary')).toBeDefined();
})



})