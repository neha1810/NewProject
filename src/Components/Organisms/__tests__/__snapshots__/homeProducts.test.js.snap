// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage render the HomePage component 1`] = `ShallowWrapper {}`;
